### ISSUE TITLE
Slack Channels and Communication for ITD Cohort

### ISSUE THREAD

**Issue by AmyCYF** [CodeYourFuture/Ops-Programme#158]
**Description**: 
Create or update Slack channels for the new ITD cohort, post introductory materials, and ensure accessibility for all participants and volunteers. Update Canvas important messages with FAQs, meeting points, and support resources.

Start Date: 20th of the previous month
Completion Date: 26th of the previous month

**Acceptance Criteria:**

- [x] ITD cohort Slack channels created or updated.
- [x] Welcome messages and key resources posted and pinned.
- [x] All participants and volunteers have access to channels.
- [x] Update Canvas
